src/explore-education-statistics-admin/src/App.tsx

```diff
@@ -42,6 +48,36 @@ function App() {
         {/* Non-Prototype Routes*/}
         <Route exact path="/admin-dashboard" component={AdminDashboardPage} />
 
+        <Route
```

Comment:
We should refactor this so that our routes and their associated links are all defined in one place and are synchronized properly.

We need something like:

```javascript
// pages/routes/editReleaseRoutes

const generateLink = section => releaseId => `/edit-release/${releaseId}/${section}`

const routes = [
  {
    path: '/edit-release/:releaseId/setup',
    component: EditReleaseSetupPage,
    title: 'Setup',
    getLink: generateLink('setup')
  },
  ...
];
```

In our routes we can do something like this:

```javascript
// App.tsx

<Router>
  {editReleaseRoutes.map(({ path, component }) => (
    <Route path={path} component={component} />
  ))}
</Router>

// OR

<Router>
  <EditReleaseRoutes />
</Router>
```

In our template, it should now be pretty straightforward to generate all of our links!

Reply:
Good suggestion.  Done, and also exported the individual route templates so that they can be used as consistent previous / next links in the various Edit Release pages.